tests/resource/test_resource_constants.py

```diff
@@ -111,6 +111,67 @@ def get_sample(sample_file: str) -> Union[str, bytes]:
 # ============================================================================
 
 TEST_RESOURCE_SET_UNIVERSAL_ACCESS_XML = get_sample("resource_set_universal_access.xml")
+# Audit Rules Request Samples
+TEST_RESOURCE_REMOVE_ALL_AUDIT_RULES_REQUEST_XML = get_sample(
+    "resource_remove_all_audit_rules.xml"
+)
+TEST_RESOURCE_OVERWRITE_AUDIT_BY_ACCESS_LEVEL_REQUEST_XML = get_sample(
+    "resource_overwrite_audit_by_access_level.xml"
+)
+TEST_RESOURCE_OVERWRITE_AUDIT_BY_ACCESS_LEVEL_MULT_REQUEST_XML = get_sample(
+    "resource_overwrite_audit_by_access_level_multiple.xml"
+)
+TEST_RESOURCE_OVERWRITE_AUDIT_BY_ACCESS_LEVEL_ALL_REQUEST_XML = get_sample(
+    "resource_overwrite_audit_by_access_level_all.xml"
+)
+TEST_RESOURCE_OVERWRITE_AUDIT_BY_ACCESS_LEVEL_NONE_REQUEST_XML = get_sample(
+    "resource_overwrite_audit_by_access_level_none.xml"
+)
+TEST_RESOURCE_ALTER_AUDIT_BY_ACCESS_LEVEL_REQUEST_XML = get_sample(
+    "resource_alter_audit_by_access_level.xml"
+)
+TEST_RESOURCE_ALTER_AUDIT_BY_ACCESS_LEVEL_MULT_REQUEST_XML = get_sample(
+    "resource_alter_audit_by_access_level_multiple.xml"
+)
+TEST_RESOURCE_ALTER_AUDIT_BY_ACCESS_LEVEL_ALL_REQUEST_XML = get_sample(
+    "resource_alter_audit_by_access_level_all.xml"
+)
+TEST_RESOURCE_ALTER_AUDIT_BY_ACCESS_LEVEL_NONE_REQUEST_XML = get_sample(
+    "resource_alter_audit_by_access_level_none.xml"
+)
+TEST_RESOURCE_OVERWRITE_AUDIT_BY_ATTEMPT_REQUEST_XML = get_sample(
+    "resource_overwrite_audit_by_attempt.xml"
+)
+TEST_RESOURCE_OVERWRITE_AUDIT_BY_ATTEMPT_MULT_REQUEST_XML = get_sample(
+    "resource_overwrite_audit_by_attempt_multiple.xml"
+)
+TEST_RESOURCE_OVERWRITE_AUDIT_BY_ATTEMPT_ALL_REQUEST_XML = get_sample(
+    "resource_overwrite_audit_by_attempt_all.xml"
+)
+TEST_RESOURCE_OVERWRITE_AUDIT_BY_ATTEMPT_NONE_REQUEST_XML = get_sample(
+    "resource_overwrite_audit_by_attempt_none.xml"
+)
+TEST_RESOURCE_ALTER_AUDIT_BY_ATTEMPT_REQUEST_XML = get_sample(
+    "resource_alter_audit_by_attempt.xml"
+)
+TEST_RESOURCE_ALTER_AUDIT_BY_ATTEMPT_MULT_REQUEST_XML = get_sample(
+    "resource_alter_audit_by_attempt_multiple.xml"
+)
+TEST_RESOURCE_ALTER_AUDIT_BY_ATTEMPT_ALL_REQUEST_XML = get_sample(
+    "resource_alter_audit_by_attempt_all.xml"
+)
+TEST_RESOURCE_ALTER_AUDIT_BY_ATTEMPT_NONE_REQUEST_XML = get_sample(
+    "resource_alter_audit_by_attempt_none.xml"
+)
+
+# ============================================================================
+# Resource Administration Getters Result Data
+# ============================================================================
+
+TEST_EXTRACT_RESOURCE_GET_AUDIT_RULES = {"success": "update", "failures": "read"}
+TEST_EXTRACT_RESOURCE_GET_AUDIT_RULES_SINGLE = {"failures": "read"}
+
+TEST_EXTRACT_RESOURCE_GET_AUDIT_RULES_WITH_ALL = {"success": "update", "all": "read"}
```

Comment:
The inclusion `EXTRACT_RESOURCE` might be a little confusing here. I would consider removing it from these samples.

There also appears to be an extraneous space here.

Reply:
Sure, taken out.